cpp/Platform.Delegates/Platform.Delegates.Delegate.h

```diff
@@ -16,54 +14,134 @@ namespace Platform::Delegates
     template <typename ReturnType, typename... Args>
     class Delegate<ReturnType(Args...)>
     {
+    public:
         using DelegateRawFunctionType = ReturnType(Args...);
         using DelegateFunctionType = std::function<DelegateRawFunctionType>;
 
+        constexpr Delegate() noexcept = default;
```

Comment:
Why we need default constructors here?

Reply:
this is an analogue of this constructor -> https://github.com/linksplatform/Delegates/blob/master/cpp/Platform.Delegates/Platform.Delegates.Delegate.h#L152

Reply:
Perfect.